modules/cpmso-feat-test-changes.adoc

```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-about.adoc
+
+:_content-type: CONCEPT
+[id="cpmso-feat-test-changes_{context}"]
+= Testing changes to the control plane configuration
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.